stable/mission-control/values.yaml

```diff
@@ -334,6 +334,8 @@ missionControl:
   internalPort: 8080
   externalPort: 80
 
+  ## Extra pre-start command to import trusted keystore, need to mount the certificates first
```

Comment:
Use a more general description. The certificate is one example you can show.
Leave the line
```
# preStartCommand:
```
Without a value. Move this example to a comment above (with leading `##`)